lib/methods/ifwapichecktask.cpp

```diff
@@ -525,7 +467,13 @@ void IfwApiCheckTask::ScriptFunc(const Checkable::Ptr& checkable, const CheckRes
 
 			Defer cancelTimeout ([&timeout]() { timeout->Cancel(); });
 
-			DoIfwNetIo(yc, checkable, cmdLine, cr, psCommand, psHost, expectedSan, psPort, *conn, *req, start);
+			DoIfwNetIo(yc, cr, psCommand, psHost, expectedSan, psPort, *conn, *req);
+
+			cr->SetExecutionEnd(Utility::GetTime());
```

Comment:
... and make this conditional.

Reply:
We already had a discussion with Julian about this and decided to put it in here, as everything that `DoIfwNetIo()` does is actually related to the check execution. Apart from that, previously the timestamp was set after shutting down the socket, so there is not much left apart from analysing the response.